docs/docs/user/references/defaults.md

```diff
@@ -1,10 +1,17 @@
 # Defaults
 
-KPOps has a very efficient way of dealing with repeating settings which manifests as [`defaults.yaml`](/resources/pipeline-defaults/defaults).
+KPOps has a very efficient way of dealing with repeating settings which manifests as [`defaults.yaml`](/resources/pipeline-defaults/defaults). This file provides the user with the power to set defaults for any and all [components](/user/references/components), thus omitting the need to repeat the same settings in [`pipeline.yaml`](/user/references/components).
 
-This file provides the user with the power to set defaults for any and all components, thus omitting the need to repeat the same settings in [`pipeline.yaml`](/user/references/components).
+??? "KPOps component Hierarchy"
+    --8<--
+    ./docs/resources/architecture/components-hierarchy.md
+    --8<--
 
-## KubernetesApp
+A very important mechanic of KPOps' defaults is that defaults set for a component apply to all components that inherit from it. An example can be found [here](#examples).
```

Comment:
consider moving the example up so that it sits right after this paragraph. jumping around a lot could make it difficult

Reply:
WDYT?

Reply:
looks good to me